src/components/TimeStamp.js

```diff
@@ -1,7 +1,7 @@
 import { DateTime } from 'luxon';
 
-const TimeStamp = (props) => {
-  const time = DateTime.fromISO(props.time);
+const TimeStamp = ({ timeStamp }) => {
+  const time = DateTime.fromISO(timeStamp);
```

Comment:
This was not necessary to change. When creating a Timestamp component, have the prop be named `time` instead of `timestamp`.